### PR TITLE
Add Exists method + test

### PR DESCRIPTION
**Pull request by fabienfoerster**
Implement the Exists() method as requested in issue #58
If it's ok then I will add examples and documentation.

### CODE REVIEW

File: firebase_test.go [hunk 1]
```diff
@@ -94,6 +94,27 @@ func TestUnauth(t *testing.T) {
...
+	t.Parallel()
+	server := firetest.New()
+	server.Start()
+	defer server.Close()
+
+	fb := New(server.URL, nil)
+	// Test Exist on an empty ref
+	b, err := fb.Exists()
+	require.NoError(t, err)
+	assert.Equal(t, false, b)
```
Comment (zabawaba99) [inline firebase_test.go]: Small nit: would you mind using `assert.False` here?

**Comment by zabawaba99** [comment]
Everything looks great @fabienfoerster, could you please add some documentation to the README?

**Comment by fabienfoerster** [comment]
Hi, I added the documentation and an example.

File: firebase_test.go (hunk 1 above)
Comment (fabienfoerster) [reply firebase_test.go]: Done :)

**Comment by zabawaba99** [comment]
Sorry it took me so long to better this @fabienfoerster ! Thanks for submitting a PR.

**Pull request merged**